codespan-lsp/Cargo.toml

```diff
@@ -16,5 +16,5 @@ codespan-reporting = { version = "0.9.5", path = "../codespan-reporting" }
 # will be valid for all the versions in this range. Getting this range wrong
 # could potentially break down-stream builds on a `cargo update`. This is an
 # absolute no-no, breaking much of what we enjoy about Cargo!
-lsp-types = ">=0.70, <0.83"
+lsp-types = "0.84"
```

Comment:
I think this would be nice if this was `>=0.84.0, <0.85` to make version range expansions easier in the future (see: https://github.com/brendanzab/codespan/pull/302/files#r532126027). :)

Reply:
It seems superfluous now, but will help dependabot in the future.